src/Api/Properties/launchSettings.json

```diff
@@ -7,7 +7,9 @@
       "applicationUrl": "http://localhost:8080",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development",
-        "ENVIRONMENT": "Local"
+        "ENVIRONMENT": "Local",
+        "AclOptions__Clients__LocalDev__Bcps__0": "bcp1",
```

Comment:
Are these here to test the formatting? As we would normally have these settings in the appsettings.Development.json file for local dev